must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -49,13 +52,19 @@ commands_desc+=("logicalvolume")
 commands_desc+=("lvmvolumegroup")
 commands_desc+=("lvmvolumegroupsnodestatus")
 commands_desc+=("pods")
+commands_desc+=("volumesnapshot -A")
+commands_desc+=("volumesnapshotclass")
```

Comment:
I believe the volumesnapshotclass is not a namespaced resource.  It should be a clusterscoped resource.

Reply:
Fixed